index.bs

```diff
@@ -975,10 +975,14 @@ the receiver decides to not attempt to load the resource at all.  If it chooses
 not to, it must respond with the appropriate failure result (such as timeout or
 invalid-url).  Additionally, the response must include the following:
 
-: state
+: state (optional)
```

Comment:
Why is the state property now optional?  The controller needs some data to maintain the local state of the media element.  That's true whether the media is flung by URL or streamed.

Reply:
I don't remember.  I'll put it back :).